src/test/java/pages/jQueryUIPages/SortablePlaceholderPage.java

```diff
@@ -0,0 +1,52 @@
+package pages.jQueryUIPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+import pages.BasePage;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public class SortablePlaceholderPage extends BasePage {
+
+    public static final String SORTABLE_PLACEHOLDER_URL = "https://jqueryui.com/sortable/#placeholder";
+    protected Logger logger = Logger.getGlobal();
```

Comment:
remove this line, you use logger from Base page

Reply:
I'm using the logger from BeaseTest in my tests, there is no logger in BasePage. That's why I decided to initiate it separately on the needed pages. Actually I can do this on BasePage and use it on pages